Gemfile

```diff
@@ -13,7 +13,8 @@ gem "unicorn-worker-killer", "0.4.4"
 
 # Federation
 
-gem "diaspora_federation-rails", "0.2.0"
+gem "diaspora_federation-json_schema", "0.2.0", github: "diaspora/diaspora_federation", branch: "develop"
```

Comment:
Version 0.2.1 is now released :)

(that also probably breaks some tests, because nil elements are now missing)